whipper/command/offset.py

```diff
@@ -85,7 +85,7 @@ def do(self):
         utils.unmount_device(device)
 
         # first get the Table Of Contents of the CD
-        t = cdrdao.ReadTOCTask(device)
+        t = cdrdao.ReadTOCTask(device)[0]
```

Comment:
If we keep the "return a tuple" behaviour for ReadTOCTask (which is a little awkward but it's not a big deal), we should write this as `t, _ = cdrdao.ReadTOCTask(device)`, which is a little less opaque than `[0]` in my opinion. Maybe a comment explaining the tocfile is unused (e.g., `...Task(device) # tocfile unused`)

Reply:
You're right, I'll improve this as suggested.